README.md

```diff
@@ -300,8 +297,6 @@ See [REFERENCE.md](https://github.com/puppetlabs/puppetlabs-inifile/blob/master/
 <a id="limitations"></a>
 ## Limitations
 
-Due to (PUP-4709) the create_ini_settings function will cause errors when attempting to create multiple ini_settings in one go when using Puppet 4.0.x or 4.1.x. If needed, the temporary fix for this can be found here: https://github.com/puppetlabs/puppetlabs-inifile/pull/196.
```

Comment:
As well as Puppet 5 being the minimum supported version, the 'temporary fix' referenced was merged years ago.